app/controllers/miq_policy_rsop_controller.rb

```diff
@@ -80,18 +80,18 @@ def rsop
   def rsop_option_changed
     assert_privileges('policy_simulation')
     if params[:event_typ]
-      @sb[:rsop][:event] = params[:event_typ] == "<Choose>" ? nil : params[:event_typ]
+      @sb[:rsop][:event] = params[:event_typ] == "" ? nil : params[:event_typ]
```

Comment:
Also, if event_type is `""`, then the condition `params[:event_typ]` will be `true`.  Instead we may want to check `if params.key?(:event_typ)`, but that has other problems because we don't want to accidentally clear the forms when the value is not passed (which is why I think they were checking for the key to begin with).

Reply:
the only time params[:event_typ] is true (ie "") is when the user selected `<Choose>`, and all other if statements don't go through because the other params (event value, filter type, filter value) will be nil regardless of what the user has them set to

Reply:
I'm not entirely sure where you were going with the `if params.key?(:event_typ)` train of thought

Reply:
Is this resolved? I think so, right?

Reply:
I'm not sure - @MelsHyrule can we hop on a call?  Maybe I'm just missing something